docker-compose.yml

```diff
@@ -67,8 +67,7 @@ services:
       - .env
       - .env.production
     volumes:
-      - app:/data/tmp/uploads
```

Comment:
Sorry I made some commits which need to be copied over to your override files. I resolved the conflict in the docker-compose.yml file, but the overrides also need this.

Reply:
We so need to mount a directory here, otherwise I cannot run imports